Source/Core/Core/NetPlayServer.cpp

```diff
@@ -861,6 +870,16 @@ void NetPlayServer::SetNetSettings(const NetSettings& settings)
   m_settings = settings;
 }
 
+bool NetPlayServer::DoAllPlayersHaveIPLDump() const
+{
+  for (const auto& p : m_players)
```

Comment:
Meant to put this in the first review, but this can be compressed to:

```cpp
return std::all_of(m_players.begin(), m_players.end(),
                   [](const auto& p) { return p.second.has_ipl_dump; });